.build-test-rules.yml

```diff
@@ -9,3 +9,11 @@ esp_delta_ota:
   disable:
     - if: SOC_WIFI_SUPPORTED != 1
       reason: Relevant only for WiFi enabled targets
+
+coap/examples/coap_server:
+  disable:
+    - if: IDF_TARGET in ["esp32h2"]
```

Comment:
In IDF, we have these examples enabled for ESP32-H2 as well. Probably for H2, the default transport goes to SPI-ethernet and hence the examples should build?